examples/nemo/scripts/kernels_trans.py

```diff
@@ -59,135 +58,76 @@
                     # to create OpenACC regions with calls to them)
                     "interp1", "interp2", "interp3", "integ_spline", "sbc_dcy",
                     "sum", "sign_"]
 
 # Routines we do not attempt to add any OpenACC to (because it breaks with
-# the PGI compiler or because it just isn't worth it)
-ACC_IGNORE = ["asm_inc_init",  # Triggers "missing branch target block"
-              "day_mth",  # Just calendar operations
-              "obs_surf_alloc",
-              "oce_alloc",
-              "trc_bc_ini",  # Str manipulation and is only an init routine
-              "trc_ini_age",  # Triggers "missing branch target block"
-              "turb_ncar",   # Resulting code seg. faults with PGI 19.4
-              "ice_dyn_adv",  # No significant compute
+# the Nvidia compiler or because it just isn't worth it)
+ACC_IGNORE = ["day_mth", # Just calendar operations
+              "obs_surf_alloc", "oce_alloc",
+              # Compiler fails w/ "Unsupported local variable"
+              # Zero performance impact since outside execution path
+              "copy_obfbdata", "merge_obfbdata",
+              "turb_ncar", # Transforming hurts performance
               "iom_open", "iom_get_123d", "iom_nf90_rp0123d",
-              "p2z_ini",  # Str manipulation in init routine
-              "p4z_ini"]  # Str manipulation in init routine
+              "trc_bc_ini", "p2z_ini", "p4z_ini"] # Str handling, init routine
 
 # Currently fparser has no way of distinguishing array accesses from
 # function calls if the symbol is imported from some other module.
 # We therefore work-around this by keeping a list of known NEMO
 # functions that must be excluded from within KERNELS regions.
-NEMO_FUNCTIONS = ["alpha_charn", "cd_neutral_10m", "cpl_freq",
-                  "cp_air",
-                  "eos_pt_from_ct",
-                  "gamma_moist",
-                  "ice_var_sshdyn", "l_vap",
-                  "sbc_dcy", "solfrac", "One_on_L",
-                  "psi_h", "psi_m", "psi_m_coare",
-                  "psi_h_coare", "psi_m_ecmwf", "psi_h_ecmwf",
-                  "q_sat", "rho_air",
-                  "visc_air", "sbc_dcy", "glob_sum",
-                  "glob_sum_full", "ptr_sj", "ptr_sjk",
-                  "interp1", "interp2", "interp3", "integ_spline"]
+NEMO_FUNCTIONS = ["alpha_charn", "cd_neutral_10m", "cpl_freq", "cp_air",
+                  "eos_pt_from_ct", "gamma_moist", "l_vap",
+                  "sbc_dcy", "solfrac", "psi_h", "psi_m", "psi_m_coare",
+                  "psi_h_coare", "psi_m_ecmwf", "psi_h_ecmwf", "q_sat",
+                  "rho_air", "visc_air", "sbc_dcy", "glob_sum",
+                  "glob_sum_full", "ptr_sj", "ptr_sjk", "interp1", "interp2",
+                  "interp3", "integ_spline"]
 
 
 class ExcludeSettings(object):
     '''
-    Class to hold settings on what to exclude from OpenACC KERNELS
-    regions.
+    Class to hold settings on what to exclude from OpenACC KERNELS regions.
```

Comment:
Since we're here, please could you update L111 so that `ExcludeSettings` does not inherit from `object`.